pyproject.toml

```diff
@@ -197,9 +197,6 @@ gettext-mo-release = { "script" = "po.build-babel:compile_mo_release" }
 translations = ["gettext-pot", "gettext-po", "gettext-mo-all"]
 icons = { "shell" = "make -C gaphor/ui/icons" }
 
-[tool.poe.executor]
-type = "virtualenv"
-
```

Comment:
Nice one, it looks like this puts the configuration back to auto which should allow for centralized virtual environments :+1: